atlasdb-api/src/test/java/com/palantir/atlasdb/keyvalue/api/CellReferenceTest.java

```diff
@@ -32,6 +32,7 @@
 import org.junit.Rule;
 import org.junit.Test;
 
+/* TODO(boyoruk): Do not forget to upgrade this to JUnit5 once we implemented new com.palantir.flake package.  */
```

Comment:
Just highlighting this so we don't forget to come back in the future. Can you create a corresponding task on Asana?